ecomstore/src/utilities/credentialUtility.py

```diff
@@ -8,7 +8,7 @@ def __init__(self):
     @staticmethod
     def get_woo_api_keys():
 
-        wc_keys = os.environ.get('WOO_KEY')
+        wc_keys = os.environ.get("WOO_KEY")
```

Comment:
This change is not needed. but it is ok. to be consistent and clean I am glad you changed it.